projects/Mallard/src/helpers/files.ts

```diff
@@ -162,15 +161,49 @@ export const getLocalIssues = async () => {
     )
 }
 
+export const getEdtionIssuesCount = async () => {
+    const editionDirList = await FSPaths.edtionDirList()
+    const regex = /\d{4}-\d{2}-\d{2}/gm // this matches issue date, i.g. 2020-02-01
+    const result: string[] = []
+    for (let i = 0; i < editionDirList.length; i++) {
+        const dir = editionDirList[i]
+        const files = await RNFS.readdir(dir)
+        const issueFiles = files.filter(file => file.match(regex) != null)
+        result.push(`${dir.split('/').pop()}: ${issueFiles.length} issues`)
+    }
+    return result
+}
+
 export const issuesToDelete = async (files: string[]) => {
     const maxAvailableEditions = await getSetting('maxAvailableEditions')
-    const edition = await getSelectedEditionSlug()
-    const lastNumberOfDays = lastNDays(maxAvailableEditions)
-    return files.filter(
-        issue =>
-            !lastNumberOfDays.map(withPathPrefix(edition)).includes(issue) &&
-            issue !== `${edition}/issues`,
-    )
+    const totalIssues = files.length
```

Comment:
Does this differentiate between editions?

Reply:
We are now generalizing issue count across editions. If the setting is to keep 7 issues, then:
for the uk, we will keep last 7 issues
for aus/us we will also keep last 7 issues, even though they are weekly issues